kuma/wiki/models.py

```diff
@@ -1584,7 +1598,10 @@ class DocumentZone(models.Model):
     attributes inherited by the topic hierarchy beneath it.
     """
     document = models.OneToOneField(Document, related_name='zone')
-    styles = models.TextField(null=True, blank=True)
+    css_filename = models.CharField(
```

Comment:
I'm chewing on the right name for this field.  For example, if you want ``zone-archive.css``, then you'd set ``css_filename`` to "zone-archive", which doesn't feel like a filename.  Maybe ``css_slug``? Maybe I'm being pedantic? 😕